tests/foreman/api/test_docker.py

```diff
@@ -1132,7 +1132,7 @@ def setUpClass(cls):
         cls.cr_external = entities.DockerComputeResource(
             name=gen_string('alpha'),
             organization=[cls.org],
-            url=settings.docker.external_url,
+            url='http://{0}:2375'.format(settings.server.hostname),
```

Comment:
We can read those values from environment variables and locally we can put on our "activate" script for virtualenvs, and on CI jenkins can export those vars. (also making them as parameters on jobs)